src/symbols/SymStore.h

```diff
@@ -40,10 +40,15 @@ class SymStore {
     SymStore(SymStore &&) = default;
     SymStore & operator=(SymStore &&) = default;
     // Constructs a new symbol.
-    SymRef newSymb(char const * fname, SRef rsort, vec<SRef> const & args, SymbolConfig const & symConfig);
+
+    SymRef newSymb(char const * fname, SRef rsort, vec<SRef> const & args, SymbolConfig const & symConfig,
```

Comment:
The public interface should stay the same.
The method with a boolean flag should be private.

Reply:
Public interface now is the same, added private method with boolean flag